src/kakarot/execution_context.cairo

```diff
@@ -59,6 +65,62 @@ namespace ExecutionContext {
             gas_used=gas_used,
             gas_limit=gas_limit,
             intrinsic_gas_cost=0,
+            starknet_address=0,
+            evm_address=0,
+            );
+        return ctx;
+    }
+
+    // @notice Initialize the execution context.
+    // @param code The code to execute.
+    // @param calldata The calldata.
+    // @return The initialized execution context.
+    func init_at_address{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        bitwise_ptr: BitwiseBuiltin*,
+    }(address: felt, calldata: felt*) -> model.ExecutionContext* {
+        alloc_locals;
+        let (empty_return_data: felt*) = alloc();
+
+        // Define initial program counter
+        let initial_pc = 0;
+        let gas_used = 0;
+        // TODO: Add support for gas limit
+        let gas_limit = 0;
+
+        let stack: model.Stack* = Stack.init();
+        let memory: model.Memory* = Memory.init();
+
+        // 1. Evm address
+        // 2. Get starknet Address
+
+        let (registry_address_) = registry_address.read();
+        let (starknet_address) = IRegistry.get_starknet_address(
+            contract_address=registry_address_, evm_address=address
+        );
+        // Get the BYTECODE from the Starknet_contract
+
+        let (code_len, code) = IEvm_Contract.code(contract_address=starknet_address);
+        // Transform byte to array?
```

Comment:
Why are we asking this question?

Reply:
good question, Danilo put this but I think it's a leftover, checking with him

Reply:
Can remove, it was just me "coding out loud"

Reply:
Let's remove it:)